src/Microsoft.PowerShell.Commands.Utility/commands/utility/group-object.cs

```diff
@@ -131,15 +165,16 @@ private static string BuildName(List<ObjectCommandPropertyValue> propValues)
                     }
                     else
                     {
-                        sb.Append(string.Format(CultureInfo.InvariantCulture, "{0}, ", propValue.PropertyValue.ToString()));
+                        sb.Append(string.Format(CultureInfo.InvariantCulture, "{0}, ", propValuePropertyValue.ToString()));
```

Comment:
Seems string.Format was optimized in .Net Core 2 but we could still use `AppendFormat()` here and in line 160 to get more readable code and exclude extra method calls.